Code/PgSQL/rdkit/adapter.cpp

```diff
@@ -381,6 +386,49 @@ extern "C" char *makeCtabText(CROMol data, int *len,
   return (char *)StringData.c_str();
 }
 
+extern "C" const char *makeMolJSON(CROMol data) {
+  std::string json = "MolToJSON not available";
+#ifdef RDK_BUILD_MOLINTERCHANGE_SUPPORT
+  auto *mol = (ROMol *)data;
```

Comment:
I noticed that pointers are never checked for non being null before dereferencing them. I guess PgSQL guarantees this may never happen.

Reply:
yeah, all of the cartridge code looks like that